### PR TITLE
freetype: enabled WOFF 2.0

### PR DESCRIPTION
**Pull request by kenmays**
WOFF File Format 2.0 - The WOFF 2.0 specification is implemented in all major browsers, and is widely used on production websites. It supports the entirety of the TrueType and OpenType specifications, including Variable fonts, Chromatic fonts, and font Collections.

### CODE REVIEW

File: media-libs/freetype/freetype-2.11.0.recipe [hunk 1]
```diff
@@ -16,7 +16,7 @@ PATCHES="
 #PATCHES="freetype-subpixel.patchset" # enable patented subpixel rendering
 
 ARCHITECTURES="all"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```
Comment (Begasus) [inline media-libs/freetype/freetype-2.11.0.recipe:19]: Not needed atm, there is no x86_gcc2 for secondary architecture (yet), but there are other recipes still mentioning it)

File: media-libs/freetype/freetype-2.11.0.recipe [hunk 2]
```diff
@@ -75,7 +78,7 @@ defineDebugInfoPackage freetype$secondaryArchSuffix \
 BUILD()
 {
 	unset meson
-	meson setup build . --buildtype release \
+	meson setup build . --buildtype release  \
```
Comment (Begasus) [inline media-libs/freetype/freetype-2.11.0.recipe:81]: Extra space added? Shouldn't be needed also.

**Review by Begasus — CHANGES_REQUESTED** [review]

File: media-libs/freetype/freetype-2.11.0.recipe (hunk 2 above)
Comment (kenmays) [reply media-libs/freetype/freetype-2.11.0.recipe]: Fixed

**Comment by korli** [comment]
Why would freetype need to encode with brotli?

**Comment by kenmays** [comment]
Moreso, to reduce web font/text latency when fonts are transferred from a web server to a web client over a network connection.

**Comment by waddlesplash** [comment]
I think we should disable this. There's not a lot of point in decoding WOFF2 directly in FreeType; WebKit decodes WOFF(2) to TTF before handing it off to the rendering engines anyway, so it can handle brotli decoding. Otherwise this will introduce another dependency to the base system.

**Pull request merged**